stream-chat-android/src/main/java/com/getstream/sdk/chat/view/MessageListView.kt

```diff
@@ -389,6 +420,34 @@ public class MessageListView : ConstraintLayout {
         setMessageListItemAdapter(adapter)
     }
 
+    public fun setLoadingView(view: View, layoutParams: FrameLayout.LayoutParams = defaultChildLayoutParams()) {
```

Comment:
Could use documentation to describe that the View passed as a parameter will be added to the View hierarchy by us, and we'll manage it. Otherwise, someone might add `MessageListView` and their own desired loading View to a layout, and then call this function, which I assume would produce an exception, since the View is already added somewhere.

Reply:
Hm, maybe kind of View factory or producing function would be better instead of passing the `View` reference here to avoid memory leaks also.

Reply:
Added docs to explain this to the user